functions/validate.py

```diff
@@ -44,5 +44,5 @@ def verify_schema_version(schema_url: str = "") -> bool:
     """
     Verify that the supplied schema is either 2.0.0, 2.0.2 or latest.
     """
-    allowed_versions = ["2.0.0", "2.0.2", "latest"]
+    allowed_versions = ["2.0.0", "2.0.2", "2.1"]
```

Comment:
Others follow SemVer, need to ensure that 2.1 isn't supposed to be referenced as 2.1.0

Reply:
We need to be sure if this list of versions are ok. 
Maybe we need to add `2.1` / `2.1.0` vs to replace `latest` with `2.1` / `2.1.0`